### PR TITLE
DBZ-6767 Add missing type name

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-6767

### CODE REVIEW

**Review by novotnyJiri — COMMENTED** [review]
The changes look good to me. But I would somehow resolve the the duplicity in the tests. Probably fix the ByteArray test so it actually tests byte array and not date array.

**Comment by jpechane** [comment]
@novotnyJiri Second round ready

**Review by novotnyJiri — APPROVED** [review]
Other than probably a naming error LGTM.

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java [hunk 1]
```diff
@@ -353,6 +353,10 @@ protected static void dropPublication() {
         dropPublication(ReplicationConnection.Builder.DEFAULT_PUBLICATION_NAME);
     }
 
+    protected static void createPublicationForAllTables() {
+        dropPublication(ReplicationConnection.Builder.DEFAULT_PUBLICATION_NAME);
```
Comment (novotnyJiri) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TestHelper.java:357]: Is it correct, that method `createPublicationForAllTables()` actually does not create any publications and just drops default publication?

**Comment by jpechane** [comment]
@novotnyJiri Thanks for the review!

**Comment by novotnyJiri** [comment]
Glad I could help!

**Pull request merged**